checkov/kubernetes/kebernetes_utils.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+from typing import Tuple, Dict, Optional, List
+
+from checkov.common.parallelizer.parallel_runner import parallel_runner
+from checkov.common.parsers.node import DictNode
+from checkov.common.runners.base_runner import filter_ignored_paths
+from checkov.kubernetes.parser.parser import parse
+
+K8_POSSIBLE_ENDINGS = [".yaml", ".yml", ".json"]
+
+
+def get_folder_definitions(
+        root_folder: str, excluded_paths: Optional[List[str]]
+) -> Tuple[Dict[str, DictNode], Dict[str, List[Tuple[int, str]]]]:
+    files_list = []
+    filepath_fn = lambda f: f'/{os.path.relpath(f, os.path.commonprefix((root_folder, f)))}'
+    for root, d_names, f_names in os.walk(root_folder):
+        filter_ignored_paths(root, d_names, excluded_paths)
+        filter_ignored_paths(root, f_names, excluded_paths)
+
+        for file in f_names:
+            file_ending = os.path.splitext(file)[1]
+            if file_ending in K8_POSSIBLE_ENDINGS:
+                full_path = os.path.join(root, file)
+                if "/." not in full_path and file not in ['package.json', 'package-lock.json']:
+                    # skip temp directories
```

Comment:
What do you mean by temp dirs?

Reply:
actually I didn't write this code, just moved it to a different file